pkg/image/prune/prune.go

```diff
@@ -1164,6 +1164,10 @@ func NewManifestDeleter() ManifestDeleter {
 }
 
 func (p *manifestDeleter) DeleteManifest(registryClient *http.Client, registryURL, repoName, manifest string) error {
-	glog.V(4).Infof("Pruning manifest for registry %q, repo %q, manifest %q", registryURL, repoName, manifest)
+	glog.V(4).Infof("Deleting manifest from registry %q: repo %q, manifest %q", registryURL, repoName, manifest)
 	return deleteFromRegistry(registryClient, fmt.Sprintf("%s/v2/%s/manifests/%s", registryURL, repoName, manifest))
 }
+
+func getName(o metav1.ObjectMeta) string {
```

Comment:
Could this take `runtime.Object` instead and access the attributes via `o.(metav1.ObjectMetaAccessor).GetObjectMeta()`?